packages/sdk/src/makeAuthenticationRpcClient.ts

```diff
@@ -31,6 +32,7 @@ export function makeAuthenticationRpcClient(
         baseUrl: url,
         interceptors: [
             ...(opts?.interceptors ?? []),
+            setHeaderInterceptor({ Version: packageJson.version }),
```

Comment:
this looks great. Now that i see this we probably want a second header which is client_id, where the different apps can optionally identify themselves for metrics purposes. that is a different task though.